x-pack/plugins/cases/public/components/all_cases/assignees_filter.test.tsx

```diff
@@ -309,4 +311,28 @@ describe('AssigneesFilterPopover', () => {
     fireEvent.change(screen.getByPlaceholderText('Search users'), { target: { value: 'dingo' } });
     expect(screen.queryByText('No assignees')).not.toBeInTheDocument();
   });
+
+  it('shows warning message when reaching maximum limit to filter', async () => {
+    const maxAssignees = Array(MAX_ASSIGNEES_FILTER_LENGTH).fill(userProfiles[0]);
+    const props = {
+      ...defaultProps,
+      selectedAssignees: maxAssignees,
+    };
+    appMockRender.render(<AssigneesFilterPopover {...props} />);
+
+    await waitFor(async () => {
+      userEvent.click(screen.getByTestId('options-filter-popover-button-assignees'));
+      expect(
+        screen.getByText(`${MAX_ASSIGNEES_FILTER_LENGTH} filters selected`)
+      ).toBeInTheDocument();
+    });
+
+    await waitForEuiPopoverOpen();
+
+    expect(
+      screen.getByText(
+        `You've selected the maximum number of ${MAX_ASSIGNEES_FILTER_LENGTH} to filter assignees`
+      )
+    ).toBeInTheDocument();
```

Comment:
Could you here and in the similar tests below also check that the elements are disabled? I guess it is enough to make sure a non-selected element is disabled and exists.